public_log_parser.php

```diff
@@ -510,6 +510,7 @@ function fill_round_zip($file, $basename, &$roundzip, $day, $round) {
 							case 'config_error.log':
 							case 'hrefs.html':
 							case 'hrefs.log':
+							case 'speech_indicators.log':
```

Comment:
you added it to disallowed logs switchcase. Only used so they don't print the warning message below